autobotAI_integrations/integrations/nessus/code_sample.py

```
@@ -1,17 +1,18 @@
+    nessus_client = context["clients"].get("nessus")
+    
+    if not nessus_client:
+        return {"error": "Nessus client not found in context. Ensure integration is configured."}
+        
+    # Example: Fetch all scans
+    response = nessus_client.list_scans()
+    
+    if response.status_code == 200:
+        scans = response.json().get("scans", [])
+        return {"scans": scans}
+    else:
```

Ruff (0.15.21)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)